### PR TITLE
changes to the iQTS page

### PR DESCRIPTION
**Pull request by Sarah-DfE**
### Trello card

https://trello.com/c/PA0ELXI4/6980-international-content-redesign-get-iqts-page

### Context
Updating the iQTS page so that the content is clear and more succinct

### CODE REVIEW

File: app/views/content/non-uk-teachers/international-qualified-teacher-status.md [hunk 1]
```diff
@@ -20,49 +20,45 @@ keywords:
...
 
 ## Why choose iQTS?
 
-iQTS is offered by world-class English teacher training providers working with schools in countries around the world. Alongside online tutorials and lectures, you’ll spend a minimum of 120 days gaining practical experience in a school in your region, and get in-person support from a trained mentor.
+iQTS is offered by teacher training providers in England that work with schools around the world. Alongside online tutorials and lectures, you’ll spend a minimum of 120 days gaining practical experience in a school in your region. You'll also get in-person support from a trained mentor.
 
-iQTS allows you to train to teach children aged 5 to 11, or children aged 11 to 18 in the full range of [secondary school subjects taught in English schools](https://www.gov.uk/national-curriculum).
+iQTS allows you to train to teach pupils aged 5 to 11, or children pupils 11 to 18 in the full range of [secondary school subjects taught in English schools](https://www.gov.uk/national-curriculum).
```
Comment (gemmadallmandfe) [inline app/views/content/non-uk-teachers/international-qualified-teacher-status.md:29]:
```suggestion
iQTS allows you to train to teach pupils aged 5 to 11, or pupils 11 to 18 in the full range of [secondary school subjects taught in English schools](https://www.gov.uk/national-curriculum).
```

**Comment by gemmadallmandfe** [comment]
@Sarah-DfE Is it worth changing the `subcategory` which appears as the blue h2 on the Non-UK page and then the iQTS `navigation_title` - which has always been quite odd in the menu?

eg change to something like
* blue h2 `If you are outside the UK`
* nav title - `Get international qualified teacher status (iQTS)`


image
image

File: app/views/content/non-uk-teachers/international-qualified-teacher-status.md [hunk 2]
```diff
@@ -127,24 +122,20 @@ The following providers specialise in particular regions, or should be contacted
 
 * [University of Portsmouth](https://www.port.ac.uk/study/short-courses-and-cpd/international-qualified-teacher-status-iqts-distance-learning) – contact this provider for more information
 
-We have recently approved the following providers to offer iQTS for the 2024/25 academic year. Their iQTS courses are still being designed – check back to provider websites for more information.
-
 * [St Mary’s University](https://www.stmarys.ac.uk/courses/postgraduate/iqts)
 
 * [University of Chichester](https://www.chi.ac.uk/)
```
Comment (Sarah-DfE) [inline app/views/content/non-uk-teachers/international-qualified-teacher-status.md:127]:
```suggestion
* [University of Chichester](https://www.chi.ac.uk/teaching/course/ipgce/)
```

**Review by Sarah-DfE — COMMENTED** [review]
updating a link

File: app/views/content/non-uk-teachers/international-qualified-teacher-status.md [hunk 3]
```diff
@@ -20,49 +20,45 @@ keywords:
...
+International qualified teacher status (iQTS) is a teaching qualification that's the equivalent of [English qualified teacher status (QTS)](/train-to-be-a-teacher/what-is-qts). iQTS enables you to train to teach in the country where you live and work, with no need to visit England.
 
 ## Why choose iQTS?
 
-iQTS is offered by world-class English teacher training providers working with schools in countries around the world. Alongside online tutorials and lectures, you’ll spend a minimum of 120 days gaining practical experience in a school in your region, and get in-person support from a trained mentor.
+iQTS is offered by teacher training providers in England that work with schools around the world. Alongside online tutorials and lectures, you’ll spend a minimum of 120 days gaining practical experience in a school in your region. You'll also get in-person support from a trained mentor.
```
Comment (emmaachesongray) [inline app/views/content/non-uk-teachers/international-qualified-teacher-status.md:27]: Should this be, 'experience in a school in your country'? We say country earlier in the sentence so should it be consistent?

File: app/views/content/non-uk-teachers/international-qualified-teacher-status.md [hunk 4]
```diff
@@ -20,49 +20,45 @@ keywords:
...
+You do not need any teaching experience or to be working in a school to apply for iQTS.
 
-* a maths qualification of the same standard as GCSE (General Certificate of Secondary Education) grade 4
+You'll need to have:
 
-* for teaching children aged 3 to 11, a science qualification of the same standard as GCSE grade 4
+* a bachelor’s degree from a UK or non-UK university (or other institution)
 
-* excellent written and verbal English language skills
+* a maths qualification of the same standard as GCSE (General Certificate of Secondary Education) grade 4 (C)
```
Comment (emmaachesongray) [inline app/views/content/non-uk-teachers/international-qualified-teacher-status.md:45]: Do you need to expand GCSE here?

File: app/views/content/non-uk-teachers/international-qualified-teacher-status.md [hunk 5]
```diff
@@ -127,24 +122,20 @@ The following providers specialise in particular regions, or should be contacted
 
 * [University of Portsmouth](https://www.port.ac.uk/study/short-courses-and-cpd/international-qualified-teacher-status-iqts-distance-learning) – contact this provider for more information
```
Comment (emmaachesongray) [inline app/views/content/non-uk-teachers/international-qualified-teacher-status.md:123]: Don't think we need the 'contact this provider for more info? As surely that statement applied to all of them?

File: app/views/content/non-uk-teachers/international-qualified-teacher-status.md [hunk 6]
```diff
@@ -155,13 +146,13 @@ If you have no teaching qualification but lots of experience, you should check y
...
+Once you have iQTS, you will be qualified to teach in England. It’s important to note that you'll also have to apply for a teaching job and a visa, unless you already have an immigration status allowing you to work in the UK.
 
-[Teach in England](/non-uk-teachers/teach-in-england-if-you-trained-overseas) gives more information.
+[Find out more about teaching in England](/non-uk-teachers/teach-in-england-if-you-trained-overseas).
 
 ## Early career teacher induction
 
-If you choose to teach in England, as an early career teacher you’ll benefit from a two-year induction period alongside your work in school. You’ll earn a salary and get mentoring and continuous professional development to help you succeed as a teacher.
+If you choose to teach in England, as an early career teacher you’ll benefit from a 2 year induction period alongside your work in school. You’ll earn a salary and get mentoring and continuous professional development to help you succeed as a teacher.
```
Comment (emmaachesongray) [inline app/views/content/non-uk-teachers/international-qualified-teacher-status.md:155]: This content is repeated twice on the page

**Review by emmaachesongray — COMMENTED** [review]
Couple of small comments

File: app/views/content/non-uk-teachers/international-qualified-teacher-status.md [hunk 7]
```diff
@@ -127,24 +122,20 @@ The following providers specialise in particular regions, or should be contacted
 
 * [University of Portsmouth](https://www.port.ac.uk/study/short-courses-and-cpd/international-qualified-teacher-status-iqts-distance-learning) – contact this provider for more information
 
-We have recently approved the following providers to offer iQTS for the 2024/25 academic year. Their iQTS courses are still being designed – check back to provider websites for more information.
-
 * [St Mary’s University](https://www.stmarys.ac.uk/courses/postgraduate/iqts)
```
Comment (gemmadallmandfe) [inline app/views/content/non-uk-teachers/international-qualified-teacher-status.md:125]: List is not in alphabetical order- should it be @Sarah-DfE ?

File: app/views/content/non-uk-teachers/international-qualified-teacher-status.md [hunk 8]
```diff
@@ -20,49 +20,45 @@ keywords:
...
 
-iQTS allows you to train to teach children aged 5 to 11, or children aged 11 to 18 in the full range of [secondary school subjects taught in English schools](https://www.gov.uk/national-curriculum).
+iQTS allows you to train to teach pupils aged 5 to 11, or pupils 11 to 18 in the full range of [secondary school subjects taught in English schools](https://www.gov.uk/national-curriculum).
 
-When you complete the course, you’ll have the skills, experience and knowledge to become an excellent teacher. You will also automatically be awarded English qualified teacher status (QTS) with no further fees or training. QTS is recognised all over the world as a teaching gold standard and will also qualify you to teach in England.
+When you complete the course, you’ll have the skills, experience and knowledge to become an excellent teacher. 
 
 If you decide to [teach in England](/non-uk-teachers/teach-in-england-if-you-trained-overseas), as an early career teacher you’ll benefit from a two-year induction period alongside your work in a school. You’ll earn a salary and get mentoring and continuous professional development to help you succeed as a teacher.
```
Comment (gemmadallmandfe) [inline app/views/content/non-uk-teachers/international-qualified-teacher-status.md:33]: This content is repeated further down the page in the `Early career teacher induction` section (as Emma points out) - can it be removed here?

File: app/views/content/non-uk-teachers/international-qualified-teacher-status.md [hunk 9]
```diff
@@ -119,32 +113,28 @@ The following providers are accepting applications from all over the world:
...
 
-* [University of Chichester](https://www.chi.ac.uk/)
+* [University of Birmingham (Dubai)](https://www.birmingham.ac.uk/dubai/study/courses/postgraduate/pgce-primary-education.aspx) – this provider offers in-person training in Dubai only, so applicants must be a commutable distance from their Dubai Campus
+
+* [University of Chichester](https://www.chi.ac.uk/teaching/course/ipgce/)
 
 * [University of Northampton](https://www.northampton.ac.uk/courses/postgraduate-certificate-in-education-international-iqts/)
 
-## Assessment
+* [University of Portsmouth](https://www.port.ac.uk/study/short-courses-and-cpd/international-qualified-teacher-status-iqts-distance-learning) – contact this provider for more information
```
Comment (Sarah-DfE) [inline app/views/content/non-uk-teachers/international-qualified-teacher-status.md:128]:
```suggestion
* [University of Portsmouth](https://www.port.ac.uk/study/short-courses-and-cpd/international-qualified-teacher-status-iqts-distance-learning)
```

**Review by gemmadallmandfe — APPROVED** [review]
Looks good to me

**Pull request merged**